### PR TITLE
Realm related tidyups

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Add missing interface methods required by Realm Provisioning.
Also fix broken CI Tests

### CODE REVIEW

File: comid/integrityregisters.go [hunk 1]
```diff
@@ -18,11 +18,11 @@ type IRegisterIndex interface{}
 
 // IntegrityRegisters holds the Integrity Registers
 type IntegrityRegisters struct {
-	m map[IRegisterIndex]Digests
+	M map[IRegisterIndex]Digests
```
Comment (setrofim) [inline comid/integrityregisters.go:21]: Why is this now being exported (this does not appear to be referenced outside the `comid` package by these changes).

If this does need to be accessible exteranlly, perhaps a more meaningful name than `M` (e.g `Digests` or `DigestMap` or something)?

Comment (yogeshbdeshpande) [reply comid/integrityregisters.go:21]: Yes, this does needs external access, hence the change.

I am not fussy about names, for brevity it was kept like that and it is a register entry, not a Digests as such!

**Review by setrofim — APPROVED** [review]
LGTM

File: comid/example_cca_realm_refval_test.go [hunk 2]
```diff
@@ -142,6 +146,22 @@ func extractRealmInstanceID(i *Instance) error {
...
+		return nil
+	}
+	b, err := r.GetBytes()
+	if err != nil {
+		return err
+	}
+	if len(b) != 64 {
+		return fmt.Errorf("invalid length %d, for realm personalization value", len(b))
+	}
+	fmt.Printf("RawValue: %x\n", b)
```
Comment (thomas-fossati) [inline comid/example_cca_realm_refval_test.go:161]: spurious printf
```suggestion
```

File: comid/example_cca_realm_refval_test.go [hunk 3]
```diff
@@ -142,6 +146,22 @@ func extractRealmInstanceID(i *Instance) error {
 	return nil
 }
 
+func extractRealmPersonalizationValue(r *RawValue) error {
+	if r == nil {
+		fmt.Println("realm persoanlization value not present")
```
Comment (thomas-fossati) [inline comid/example_cca_realm_refval_test.go:151]: spurious printf
```suggestion
```

File: comid/classid_test.go [hunk 4]
```diff
@@ -201,6 +201,93 @@ func TestClassID_SetOID_ok(t *testing.T) {
...
+		{
+			desc:        "invalid format, no digits",
+			input:       "1...",
+			expectedErr: `invalid OID: strconv.Atoi: parsing "": invalid syntax`,
+		},
+	}
+	for _, tv := range tvs {
+		c := &ClassID{}
+		err := c.SetOID(tv.input)
+		fmt.Printf("Error = %s", err.Error())
```
Comment (thomas-fossati) [inline comid/classid_test.go:286]: spurious printf
```suggestion
```

**Review by thomas-fossati — APPROVED** [review]
LGTM

(before merging please remove the extra printf's)

**Pull request merged**